bin/pygrb/pycbc_pygrb_results_workflow

```diff
@@ -273,9 +280,15 @@ plot_node, skygrid_plot = _workflow.make_pygrb_plot(
     wflow, 'pygrb_plot_skygrid', out_dir, trig_file=offsource_file
 )
 plotting_nodes.append(plot_node)
+# Plot the template bank
+plot_node, bank_plot = _workflow.make_pygrb_plot(
+    wflow, 'pycbc_plot_bank_corner', out_dir, bank_file=bank_file
+)
+
```

Comment:
I think you also need to append `plot_node` to `plotting_nodes`, as done in the lines above for the sky grid plot.

Reply:
I guess in your test, the job may have completed early enough to hide the missing dependency. But we might not always be so lucky [1], so it best to do this properly.

[1] https://en.wikipedia.org/wiki/Race_condition

Reply:
Done.